File: week2/day4/week2-day4-workshop/week2-day4-workshop/Mentor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace week2_day4_workshop
{
    class Mentor:Person
    {
        public string Level { get; set; }

        public Mentor(string name,int age,string gender,string level)
        {
            this.Name = name;
            this.Age = age;
            this.Gender = gender;
            this.Level = level;
        }

        public Mentor():this("Jane Doe",30,"female","intermediate")
        {
            this.Level = "intermediate";
        }

        public override void GetGoal()
        {
            Console.WriteLine("My goal is: Educate brilliant junior software developers.");
        }

        public override void Introduce()
        {
            Console.WriteLine($"Hi, I'm {this.Name}, a {this.Age} year old {this.Gender} {this.Level} mentor.");
        }

    }
}

File: DSA-2019/week-02/day-1/mentor.py
from person import Person


class Mentor(Person):
    def __init__(self, name="Jane Doe", age="30", gender="femalw", level="intermediate"):
        Person.__init__(self, name, age, gender)
        self.level = level

    def get_goal(self):
        return "Educate brilliant junior software developers."

    def introduce(self):
        return f"I'm {self.name}, a {self.age} year old {self.gender} {self.level} mentor."
